qcelemental/models/basemodels.py

```diff
@@ -118,3 +118,6 @@ def msgpack(self,
     @classmethod
     def _parse_msgpack(cls, data: bytes) -> Dict[str, Any]:
         return msgpack_loads(data)
+
+    def to_string(self):
```

Comment:
This overloads the built in `to_string` method of BaseModel. I'm okay with that, but it does have a different call signature, which may cause some confusion if someone tries to use it

Reply:
I think its something that we can add back. The short is that the strings were not very useful  in our context so we needed to stop them as they cost a prodigious amount co compute. We often overload this individually in inherited projects.